inst/rmd/h/default.html

```diff
@@ -602,7 +602,7 @@ <h2 id="$idprefix$toc-title">$toc-title$</h2>
 
 // add bootstrap table styles to pandoc tables
 function bootstrapStylePandocTables() {
-  $$('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
+  $$('tr.header').parent('thead').parent('table').addClass('table table-condensed');
```

Comment:
How about finding tables that have either `thead > tr.header` or `tbody > tr.odd` as their children, and add the classes to them? Would that fix both problems?